doc/user/content/sql/isolation-level.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Isolation Level"
+description: "The isolation level defines how Materialize isolates the execution of transactions."
+aliases:
+ - /sql/consistency
+menu:
+  main:
+    parent: 'reference'
+    weight: 160
+---
+
+The SQL standard defines four levels of transaction isolation. In order of least strict to most strict they are:
+
+  * Read Uncommitted
+  * Read Committed
+  * Repeatable Read
+  * Serializable
+
+In Materialize, you can request any of these isolation
+levels, but they all behave the same as the Serializable isolation level. In addition to the four levels defined in the
+SQL Standard, Materialize also defines a [Strict Serializable](#strict-serializable) isolation level.
+
+Isolation level is a per session configurable variable. To set the current session’s isolation level you can
+execute `SET TRANSACTION_ISOLATION TO '<isolation-level>';`. The default isolation level is Strict Serializable.
+
+## Serializable
+
+The SQL standard defines the Serializable isolation level as preventing the following three phenomenons:
+
+- **P1 (”Dirty Read”):**
+  > SQL-transaction T1 modifies a row. SQL-transaction T2 then reads that row before T1 performs a
+  COMMIT. If T1 then performs a ROLLBACK, T2 will have read a row that was never committed and that may thus be
+  considered to have never existed.
+
+- **P2 (”Non-repeatable read”):**
+
+  > SQL-transaction T1 reads a row. SQL-transaction T2 then modifies or deletes that row and performs
+  a COMMIT. If T1 then attempts to reread the row, it may receive the modified value or discover that the row has been
+  deleted.
```

Comment:
If this is a quote, no problem, but the words here are confusing to me. Is the idea that `T1` and `T2` are both "in progress" and neither has committed yet?

Reply:
I guess that is the only way that makes sense, but .. if it's not a quote I would be tempted to say "in-progress SQL-transaction T1..." or the like.

Reply:
This is a quote, it's copied and pasted directly from the SQL standard. Though if we think the wording is confusing, then we can replace it with our own words/interpretation.